src/RetireNet.Packages.Tool/Services/RetireServiceOptions.cs

```diff
@@ -4,10 +4,15 @@ namespace RetireNet.Packages.Tool.Services
 {
     public class RetireServiceOptions
     {
+        private string path;
 
         public Uri RootUrl { get; set; }
 
-        public string Path { get; set; }
+        public string Path
+        {
+            get => path;
+            set => path = value?.Replace('/', System.IO.Path.DirectorySeparatorChar).Replace('\\', System.IO.Path.DirectorySeparatorChar);
```

Comment:
What scenario does this handle?

Reply:
If I can, I like to avoid logic in getters and setters. Is it possible to move this logic to where the Path getter is in use, or would it involve code duplication..?

Reply:
It makes sure, that every `path` you specify, even `<I/am\a/path>` gets transformed into a consistent path, e.g. linux `<I/am/a/path>`. 

Also, if you would use a consistent `<I/am/a/path>`, the `FileService` would concat `<I/am/a/path\obj\app.settings.json` in windows environments.

So with this specialized setter, every path is transformed into the correct separator

Windows: `<I\am\a\path\obj\app.settings.json`
Linux: `<I/am/a/path/obj/app.settings.json`

In my opinion this is more readable as mixed separators, but of course this is not a must-have, everything should work as expected without. If you don't see this in the master branch, I can quickly revert this change.

Reply:
> If I can, I like to avoid logic in getters and setters. Is it possible to move this logic to where the Path getter is in use, or would it involve code duplication..?

I absolutely agreee with you, to avoid logic from `dumb` classes :) I will check, if I can move it.

Reply:
I moved the logic to the direct and only initialization of the `Path` property from the outside in the `Host` class. This should make sure, that everything is still working as expected and the path is transformed.